src/Chirp.Razor/Pages/Public.cshtml.cs

```diff
@@ -1,7 +1,14 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using Chirp.Core;
-using Chirp.Infrastructure;
+
+/*
+<Summary>
+This is model for the public page
```

Comment:
I believe there is missing a 'the' on line 7.

Maybe also add that the razor pages links it with the Public.cshtml file